lib/node_modules/@stdlib/array/base/count-ifs/README.md

```diff
@@ -0,0 +1,149 @@
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+-->
+
+# countIfs
+
+> Count the number of elements in the input arrays which pass the tests implemented by the predicate functions.
+
+<!-- Section to include introductory text. Make sure to keep an empty line after the intro `section` element and another before the `/section` close. -->
+
+<section class="intro">
+
+</section>
+
+<!-- /.intro -->
+
+<!-- Package usage documentation. -->
+
+<section class="usage">
+
+## Usage
+
+```javascript
+var countIfs = require( '@stdlib/array/base/count-ifs' );
+```
+
+#### countIfs( x0, predicate0\[, x1, predicate1\[, x2, predicate2, ...]] )
+
+Counts the number of elements in the input arrays which pass the tests implemented by the predicate functions.
+
+```javascript
+function predicate0( value ) {
+    return ( value > 0 );
+}
+
+function predicate1( value ) {
+    return ( value < 3 );
+}
+
+var x0 = [ 0, 1, 0, 1, 2 ];
+var x1 = [ 2, 3, 1, 2, 5 ];
+
+var out = countIfs( x0, predicate0, x1, predicate1 );
+// returns 1
+```
+
+The `predicate` function is provided three arguments:
+
+-   **value**: current array element.
+-   **index**: current array element index.
+-   **arr**: input array.
+
+</section>
+
+<!-- /.usage -->
+
+<!-- Package usage notes. Make sure to keep an empty line after the `section` element and another before the `/section` close. -->
+
+<section class="notes">
+
+## Notes
+
+-   The function assumes that all input arrays have the same length.
+-   The function supports array-like objects supporting the accessor protocol (e.g., [`Complex128Array`][@stdlib/array/complex128], [`Complex64Array`][@stdlib/array/complex64], etc).
+
+</section>
+
+<!-- /.notes -->
+
+<!-- Package usage examples. -->
+
+<section class="examples">
+
+## Examples
+
+<!-- eslint no-undef: "error" -->
+
+```javascript
+var discreteUniform = require( '@stdlib/random/array/discrete-uniform' );
+var isPositiveInteger = require( '@stdlib/assert/is-positive-integer' ).isPrimitive;
+var isNegativeInteger = require( '@stdlib/assert/is-negative-integer' ).isPrimitive;
+var naryFunction = require( '@stdlib/utils/nary-function' );
+var countIfs = require( '@stdlib/array/base/count-ifs' );
+
+var x = discreteUniform( 10, -5, 5, {
+    'dtype': 'int32'
+});
+// returns <Int32Array>
+
+var y = discreteUniform( 10, -5, 5, {
+    'dtype': 'int32'
+});
+// returns <Int32Array>
+
+var out = countIfs( x, naryFunction( isPositiveInteger, 1 ), y, naryFunction( isNegativeInteger, 1 ) ); // eslint-disable-line max-len
```

Comment:
You need to disable lint rules via an HTML comment placed before this code block. See other READMEs.